test/main.js

```diff
@@ -2,67 +2,90 @@ const request = require('supertest');
 const app = process.env.PROXY_URL;
 const { expect } = require('chai');
 
-describe('Main Site', () => {
-  it('should thro 404 b/c no index.html found', () => {
-    request(app)
-      .get('/')
-      .expect(404)
-      .then((response) => {
-        expect(response.text.indexOf('Page not found')).to.be.above(-1);
-      });
-  });
-
-  it('404.html found', () => {
-    request(app)
-      .get('/404.html')
+describe('robots.txt', () => {
```

Comment:
Tests which depended on the content of upstream servers were removed in favor os just testing the functionality of the proxy itself. Tried to be as semantic as possible with the new ones.